.github/workflows/publish.yml

```diff
@@ -2,6 +2,8 @@ on:
   workflow_dispatch:
   push:
     branches: main
+  pull_request:
```

Comment:
@catalamarti wouldn't this means the book gets published for every open pull request? I think you would only want it updated once on main, no?